antarest/tools/lib.py

```diff
@@ -207,10 +208,7 @@ def extract_commands(study_path: Path, commands_output_dir: Path) -> None:
     command_list = extractor.extract(study)
 
     (commands_output_dir / COMMAND_FILE).write_text(
-        json.dumps(
-            [command.model_dump(exclude={"id"}) for command in command_list],
-            indent=2,
-        )
+        to_json([command.model_dump(exclude={"id"}) for command in command_list], indent=2).decode("utf-8")
```

Comment:
we can provide 2 methods, one to get bytes, the other to get strings, to not have to repeat the `decode` everywhere we need a string